src/components/ReportModal/ReportModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { useNote } from 'components/NoteDataProvider';
+import ReportSection, { PhysicianNotesSection } from './ReportSection';
+import styles from './ReportModal.module.scss';
+import { faUser, faNotesMedical, faRoute, faStickyNote } from '@fortawesome/free-solid-svg-icons';
+import ActionButton from 'components/ActionButton';
+
+interface ReportModalInterface {
+  onConfirm: () => void;
+  onDecline: () => void;
+}
+
+const ReportModal: FC<ReportModalInterface> = ({ onConfirm, onDecline }) => {
+  const note = useNote();
+  const reportName = `${note?.pathway} Report`;
```

Comment:
Up for discussion since I know the design says "{Pathway name} Report" but I think this should be "{State name} Report" since we are submitting a note to accept a specific state

Reply:
That makes more sense to me.  @dehall Thoughts? ☕️

Reply:
TBH both feel a little awkward to me though I won't strongly oppose either. Not to bikeshed but personally I would lean toward just "Pathway Report". The pathway and state name should both be included here anyway right?

Reply:
Changed the name to "Pathway Report"